packages/sn-redux/src/reducers/sharing.ts

```diff
@@ -0,0 +1,35 @@
+import { combineReducers, Reducer } from 'redux'
+import { SharingLevel, SharingMode } from '@sensenet/client-core'
+
+export interface SharingEntry {
```

Comment:
This interface is the same as ODataSharingResponse from client-core. I think it would make sense to use that.